ide/winvs/README.md

```diff
@@ -0,0 +1,28 @@
+VisualStudio solution for wolfCLU
+=================================
+
+The solution file, wolfclu.sln, facilitates bulding wolfCLU.
+The solution provides both Debug and Release
+builds of Dynamic 32- or 64-bit libraries. The file
+`user_settings.h` should be used in the wolfSSL build to configure it.
+
+The file `wolfclu\ide\winvs\user_settings.h` contains the settings used to
+configure wolfSSL with the appropriate settings. This file must be copied
+from the directory `wolfclu\ide\winvs` to `wolfssl\IDE\WIN`. The file
+`wolfclu\ide\winvs\options.h` should also be copied to 'wolfssl/wolfssl`. You
+can then build wolfSSL with support for wolfCLU. 
+
+After building the wolfSSL solution, copy the resulting `wolfssl.lib`
+found in either the `Debug` or `Release` directory, into the wolfclu
+directory.
```

Comment:
I just thought that including all the possible directories (release/x64, debug/x32) would break wolfCLU if it found the wrong wolfssl.lib built with a different architecture. That's only an issue if you build wolfSSL more than once with different configurations though

Reply:
switching to the different build options of wolfCLU should have different directories that it searches for wolfssl.lib in

Reply:
When making the change to the solutions properties, select the desired Configuration: and Platform: by default they are set to the active build. For each build x64 debug/release, Win32, they can have different directories set.

Reply:
Done.

Reply:
Am still seeing it not look for the wolfssl.lib from the wolfssl directory. This would be adjusting the properties->Linker->General->Additional Library Directories and setting it to the default directory that wolfSSL generates the library in.

Reply:
Apologies. Overwrote the version I was trying to push.